### ISSUE TITLE
Two failures during compilation on Raspberry Pi

### ISSUE THREAD

**Issue by holybladder** [badaix/snapcast#1134]
**Describe the bug**
Compilation fails at two steps during installation on Raspberry Pi when following instructions at https://github.com/badaix/snapcast/blob/develop/doc/build.md#raspberry-pi-cross-compile
(was able to resolve the first but not the second)

**Describe the bug**
Makefiles not initially present, 

**Steps to Reproduce**
1. Checked that git and all dependencies were installed
2. Ran "git clone https://github.com/badaix/snapcast.git"
3. "cd snapcast"
4. "make" - Make fails because makefiles not present
5. Ran "git checkout master" - this checked out makefiles (saw it on a forum, don't know enough to understand why this worked, sorry)
6. "make" - make now found makefiles and ran through correctly
7. "make installclient" - this failed with the following output:

make install -C client 
make[1]: Entering directory '/home/pi/New/snapcast/client'
make adduser
make[2]: Entering directory '/home/pi/New/snapcast/client'
sh ../debian/snapclient.postinst configure
sh: 0: cannot open ../debian/snapclient.postinst: No such file
make[2]: *** [Makefile:181: adduser] Error 2
make[2]: Leaving directory '/home/pi/New/snapcast/client'
make[1]: *** [Makefile:145: install] Error 2
make[1]: Leaving directory '/home/pi/New/snapcast/client'
make: *** [Makefile:22: installclient] Error 2

**Environment details**
Raspberry Pi 3B+ running fresh Pi OS Bullseye installation
Snapcast straight from git

**Attach logfile if applicable**
No log possible, install failed

**Comment by kingosticks** [comment]
The wiki has been outdated for some time but I believe it's now `cmake` rather than `make`. You can see this working in the GH Actions logs at https://github.com/badaix/snapcast/actions

**Comment by holybladder** [comment]
> The wiki has been outdated for some time but I believe it's now `cmake` rather than `make`. You can see this working in the GH Actions logs at https://github.com/badaix/snapcast/actions

Oh, I see. Okay, I ran "cmake ." then "cmake --install" but got this:
```
-- Install configuration: ""
-- Up-to-date: /usr/local/bin/snapserver
-- Up-to-date: /usr/local/share/man/man1/snapserver.1
-- Up-to-date: /usr/local/etc/snapserver.conf
-- Up-to-date: /usr/local/share/snapserver/index.html
-- Up-to-date: /usr/local/share/snapserver/snapweb
-- Up-to-date: /usr/local/share/snapserver/snapweb/stop.png
-- Up-to-date: /usr/local/share/snapserver/snapweb/speaker_icon.png
-- Up-to-date: /usr/local/share/snapserver/snapweb/manifest.json
-- Up-to-date: /usr/local/share/snapserver/snapweb/styles.css
-- Up-to-date: /usr/local/share/snapserver/snapweb/3rd-party
-- Up-to-date: /usr/local/share/snapserver/snapweb/3rd-party/libflac.js
-- Up-to-date: /usr/local/share/snapserver/snapweb/config.js
-- Up-to-date: /usr/local/share/snapserver/snapweb/mute_icon.png
-- Up-to-date: /usr/local/share/snapserver/snapweb/play.png
-- Up-to-date: /usr/local/share/snapserver/snapweb/10-seconds-of-silence.mp3
-- Up-to-date: /usr/local/share/snapserver/snapweb/snapcast-512.png
-- Up-to-date: /usr/local/share/snapserver/snapweb/index.html
-- Up-to-date: /usr/local/share/snapserver/snapweb/snapstream.js
-- Up-to-date: /usr/local/share/snapserver/snapweb/launcher-icon.png
-- Up-to-date: /usr/local/share/snapserver/snapweb/favicon.ico
-- Up-to-date: /usr/local/share/snapserver/snapweb/snapcontrol.js
-- Up-to-date: /usr/local/share/snapserver/plug-ins/meta_mpd.py
-- Up-to-date: /usr/local/share/snapserver/plug-ins/meta_mopidy.py
-- Up-to-date: /usr/local/share/snapserver/plug-ins/meta_librespot-java.py
CMake Error at client/cmake_install.cmake:52 (file):
  file INSTALL cannot find "/home/pi/New/snapcast/bin/snapclient": No such
  file or directory.
Call Stack (most recent call first):
  cmake_install.cmake:49 (include)
```

**Comment by badaix** [comment]
Documentation is updated: 18c4a986a68a9b58bf2bd50479cdfeb8a8534090

**Comment by badaix** [comment]
Fixed in version [0.28.0](https://github.com/badaix/snapcast/releases/tag/v0.28.0)
